nativescript-angular/view-util.ts

```diff
@@ -249,8 +264,11 @@ export class ViewUtil {
             this.removeLayoutChild(parent, child);
         } else if (isContentView(parent) && parent.content === child) {
             parent.content = null;
-            parent.lastChild = null;
-            parent.firstChild = null;
+        } else if (child.nodeName === "DetachedContainer") {
+            // Skip - DetachedContainer is... well detached from its parent
+            // Used with ListViews and other TemplatedItemsComponent views.
+        } else if (child.nodeName === "FormattedString" || child.nodeName === "Span") {
+            // Removing FormattedString and its Spans from a Label throws an exception
```

Comment:
Let's try to address these crashes in a separate issue/PR. I don't think it is a good practice to have element specific code inside the renderer.